git-tar/function/ops.go

```diff
@@ -316,91 +317,98 @@ func deploy(tars []tarEntry, pushEvent sdk.PushEvent, stack *stack.Services, sta
 	c := http.Client{}
 	gatewayURL := os.Getenv("gateway_url")
 
+	sem := make(chan int, 4)
+	eg, _ := errgroup.WithContext(context.Background())
+
 	for _, tarEntry := range tars {
-		fmt.Println("Deploying service - " + tarEntry.functionName)
+		sem <- 1
+		eg.Go(func() error {
```

Comment:
This is too nested now, I'd like to see it broken down and broken out.

Reply:
It's addressed on next lines